docs/src/installation.md

```diff
@@ -42,6 +42,7 @@ is often more complex. We list below the currently available solvers.
 | [Ipopt](https://projects.coin-or.org/Ipopt)                                    | [Ipopt.jl](https://github.com/JuliaOpt/Ipopt.jl)                                 | EPL     | LP, QP, NLP                        |
 | [MOSEK](http://www.mosek.com/)                                                 | [MosekTools.jl](https://github.com/JuliaOpt/MosekTools.jl)                       | Comm.   | LP, MILP, SOCP, MISOCP, SDP        |
 | [OSQP](https://osqp.org/)                                                      | [OSQP.jl](https://github.com/oxfordcontrol/OSQP.jl)                              | Apache  | LP, QP                             |
+| [ProxSDP](https://github.com/mariohsouto/ProxSDP.jl)                                | [ProxSDP.jl](https://github.com/mariohsouto/ProxSDP.jl)                     | MIT     | LP, SOCP, SDP                      |
```

Comment:
Markdown is very picky about the formatting of tables. It's hard to tell from the diff if the whitespace is correct. Did you try [building the docs](https://github.com/JuliaOpt/JuMP.jl/tree/master/docs#building-the-documentation) to confirm that the table build correctly?

Reply:
I have tried previewing in vs code but building is indeed more guaranteed

![image](https://user-images.githubusercontent.com/32756941/55121382-098ef580-50d9-11e9-8911-334f18b6554d.png)